README.md

```diff
@@ -229,26 +229,26 @@ As work in progress, there are plently of issues and features missing, which we
+| [cpp/jvm&#x2011;enclave&#x2011;common/](cpp/jvm-enclave-common)                                                                                                                                                                                    | Implementations and stubs of Linux system POSIX calls inside the enclave.                                        |
+| [cpp/jvm&#x2011;host/](cpp/jvm-host)                                                                                                                                                                                                               | Native code which interacts with the Java/Kotlin layer through JNI.                                              |
+| [cpp/jvm&#x2011;host&#x2011;enclave&#x2011;common/](cpp/jvm-host-enclave-common)                                                                                                                                                                   | Native code used both by the host and the enclave. It mainly consists of utility functions/classes.              |
+| [cpp/linux&#x2011;sgx/](cpp/linux-sgx)                                                                                                                                                                                                             | Conclave modifications to the [Intel SGX SDK](https://github.com/intel/linux-sgx).                               |
+| [cpp/substratevm/](cpp/substratevm)                                                                                                                                                                                                                | C/C++ enclave code, with the implementation of the entry points (host to enclave) of some EDL code.              |
+| [docs/](docs)                                                                                                                                                                                                                                      | Source code for the [Conclave documentation](https://docs.conclave.net).                                         |
+| [integration&#x2011;tests/](integration-tests)                                                                                                                                                                                                     | A separate independent Gradle project which is used to test the SDK artifacts at an integration-level.           |
+| [plugin&#x2011;enclave&#x2011;gradle/](plugin-enclave-gradle)                                                                                                                                                                                      | The Conclave Gradle enclave plugin which automates the process of building a native SGX binary.                  |
```

Comment:
Please add a comma after `The Conclave Gradle enclave plugin`.

Reply:
Done.